src/App.jsx

```diff
@@ -1,7 +1,14 @@
-export default function App() {
+import JokeCard from "./components/JokeCard";
+import jokes from "./data/jokes";
+
+const App = () => {
   return (
-    <div className="App">
-      <h1 className="text-2xl text-red-500">Hello React!</h1>
-    </div>
+    <section className="flex justify-center items-center h-screen bg-[#e0f7dd] gap-10">
+      {jokes.map((joke) => (
+        <JokeCard {...joke} key={joke.id} />
+      ))}
```

Comment:
Inside the map, since you have only one argument you can commit the parenthesis, also you can make this function one line since you're returning one expression, it is more readable that way. 
```suggestion
      {jokes.map(joke => (<JokeCard {...joke} key={joke.id} />))}
```

Reply:
That was what I wanted as well but sometimes the code formatter (Prettier) automatically formats the code in an opinionated way and adds the parenthesis or puts code on separate lines even if you removed them.

But thankfully I was able to commit your suggestion directly from GitHub I think now it is well formatted